lua/legendary/compat/which-key.lua

```diff
@@ -17,7 +17,7 @@ end
 ---@param which_key_opts table
 ---@return LegendaryItem[]
 function M.parse_whichkey(which_key_tbls, which_key_opts)
-  local wk_parsed = require('which-key.keys').parse_mappings({}, which_key_tbls, which_key_opts.prefix or '')
+  local wk_parsed = require('which-key.keys').parse_mappings({}, which_key_tbls, which_key_opts and which_key_opts.prefix or '')
```

Comment:
We're failing `luacheck` now because the line is too long. Just needs reformatted:

```suggestion
  local wk_parsed = require('which-key.keys').parse_mappings(
  {},
  which_key_tbls,
  which_key_opts and which_key_opts.prefix or ''
)
```